openstudiocore/src/cli/openstudio_cli.rb

```diff
@@ -294,17 +293,6 @@ def parse_main_args(main_args)
         #$logger.warn "'#{dir}' passed to #{main_args[i]} is not a directory"
       end
       new_path << dir
-    elsif main_args[i] == '--no-ssl'
```

Comment:
@axelstudios should probably still handle this argument and just spit out a message that it is no longer needed for backwards compatibility.  I know residential was using this.

Reply:
No big deal, we can update our code to not pass this argument.